### ISSUE TITLE
[JungleBus] implement support for editing relations associated to a node

### ISSUE THREAD

**Issue by overflorian** [jawg/osm-contributor#91]
Child issue of #89

**Comment by overflorian** [comment]
Warning:
there are two kind of lines:
-One direction. Only a master_relation is available in OSM
e.g. https://www.openstreetmap.org/relation/7342872
-Multi-direction lines (most frequent case). A master_relation embed 2 child relations, one per direction.
In this case, only the child relations need to be taken into account.
e.g. http://www.openstreetmap.org/relation/954977

**Comment by nlehuby** [comment]
No, the relation route_master is supposed to have only route relations in it.
"The route master is a relation that contains all the direction and variant routes"
https://wiki.openstreetmap.org/wiki/Proposed_features/Public_Transport#Route_master

So you only have to deal with route relations.
